### PR TITLE
last_active flag for share_limits

### LINKED ISSUE #353: [FR]: Add a time_inactive to the share_limits
### Is your feature request related to a problem? Please elaborate.

Sometimes I wouldn't like to stop a torrent if it actively seeding everyday. With this option, I could say if a torrent has reached its seeding goals but people are still downloading, it should be kept online. It helps people and helps to maintain a good ratio.

### Describe the solution you'd like

Another flag option in `share_limits` that will keep the torrent online if there was activity in the last X minutes.

### Does your solution involve any of the following?

- [X] New config option
- [ ] New command option

### Describe alternatives you've considered

I currently use `autoremove-torrents` to achieve this functionality. But `qbit_manage` has added more granular control for the seeding times, almost rendering the former useless. For me personally, this issue is holding me back.

Example of a config in `autoremove-torrents`:

`remove: seeding_time > 259200 AND last_activity > 259200`

### Who will this benefit?

People who care about seeding and people downloading the torrents. Also some niche group that uses `autoremove-torrents` maybe? I don't know about that! :)

### Additional Information

_No response_

### PR DESCRIPTION
**Pull request by estebanthi**
## Description

Added a `last_active` flag for `share_limits` to resume torrents and avoid cleanup if there was activity in the last X minutes.

Fixes #353 (issue)

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)
- [x] This change requires a documentation update

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have added or updated the docstring for new or existing methods
- [x] I have modified this PR to merge to the develop branch

### CODE REVIEW

File: modules/core/share_limits.py [hunk 1]
```diff
@@ -210,7 +213,7 @@ def update_share_limits_for_group(self, group_name, group_config, torrents):
                 check_max_ratio or check_max_seeding_time or check_limit_upload_speed or share_limits_not_yet_tagged
             ) and hash_not_prev_checked:
                 if not is_tag_in_torrent(MIN_SEEDING_TIME_TAG, torrent.tags) and not is_tag_in_torrent(
-                    MIN_NUM_SEEDS_TAG, torrent.tags
+                    MIN_NUM_SEEDS_TAG, torrent.tags and not is_tag_in_torrent(LAST_ACTIVE_TAG, torrent.tags)
                 ):
```
Comment (bobokun) [inline modules/core/share_limits.py:217]: Isn't the ending bracket misplaced?


```suggestion
                if not is_tag_in_torrent(MIN_SEEDING_TIME_TAG, torrent.tags) and not is_tag_in_torrent(
                    MIN_NUM_SEEDS_TAG, torrent.tags) and not is_tag_in_torrent(LAST_ACTIVE_TAG, torrent.tags
                ):
```

**Review by bobokun — COMMENTED** [review]
Has this been tested locally yet and confirmed working?

File: modules/core/share_limits.py (hunk 1 above)
Comment (estebanthi) [reply modules/core/share_limits.py:217]: Good point, it is.

**Comment by estebanthi** [comment]
Yes, I'm using it since the PR is opened and it's working. Strangely, the misplaced bracket you noticed doesn't seem to be causing any problems.

**Comment by bobokun** [comment]
Similar to the other PR can you please bump the version and update the `config.yml.sample` with comments + an example

**Comment by estebanthi** [comment]
Done.

**Review by bobokun — APPROVED** [review]

**Pull request merged**